### PR TITLE
Update to released Phoenix 1.3

### PR DESCRIPTION
**Pull request by ryanlntn**
It seems the file structure has changed since this was originally spun up. I took a crack at updating to the new file structure.

### CODE REVIEW

File: template/$PROJECT_NAME$/bin/ci [hunk 1]
```diff
@@ -14,4 +14,4 @@ cd assets/ && {
 MIX_ENV=test mix compile --warnings-as-errors --force || { echo 'Please fix all compiler warnings.'; exit 1; }
 MIX_ENV=test mix credo --strict --ignore design,consistency || { echo 'Elixir code failed Credo linting. See warnings above.'; exit 1; }
 MIX_ENV=test mix docs  || { echo 'Elixir HTML docs were not generated!'; exit 1; }
-mix test || { echo 'Elixir tests failed!'; exit 1; }
+mix test || { echo 'Elixir tests failed!'; exit 1; }
```
Comment (jamonholmgren) [inline template/$PROJECT_NAME$/bin/ci]: Removing trailing line breaks?

**Review by jamonholmgren — APPROVED** [review]
Looks great!

File: template/$PROJECT_NAME$/bin/ci (hunk 1 above)
Comment (ryanlntn) [inline template/$PROJECT_NAME$/bin/ci]: Yeah my editor cleans all the things.

Comment (jamonholmgren) [inline template/$PROJECT_NAME$/bin/ci]: I thought we left trailing line breaks in.

**Comment by jamonholmgren** [comment]
Any idea why CI is failing?

**Comment by ryanlntn** [comment]
Fixed it. I broke file permissions on the bin scripts. Adding a couple more tweaks as well.

File: template/$PROJECT_NAME$/bin/ci (hunk 1 above)
Comment (ryanlntn) [inline template/$PROJECT_NAME$/bin/ci]: Sounds sloppy.

**Pull request merged**